lib/diff.ml

```diff
@@ -35,6 +39,11 @@ let extract_items items =
           Sig_item_map.add (Module_item, Ident.name id) (Mod mod_decl) tbl
       | Sig_value (id, val_des, _) ->
           Sig_item_map.add (Value_item, Ident.name id) (Val val_des) tbl
+      | Sig_class_type (id, class_type_decl, _, _) ->
+          (*Extended to include Class_type_declaration*)
```

Comment:
Can you please remove this comment as it does not bring useful information and will age poorly.